libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -442,6 +540,46 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
         });
     };
 
+    // unit test
```

Comment:
Is this a TODO note? I have not found the mentioned unit test anywhere..

Reply:
Sorry, forgotten here. I've added unit tests that check the generated scheduled mail object